.github/workflows/run_helm_tests.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
         done
     - name: Setup 18083 port forwarding
       run: |
-        nohup kubectl port-forward service/${EMQX_NAME} 18083:18083 > /dev/null &
+        nohup bash -c "while true; do kubectl port-forward service/${EMQX_NAME} 18083:18083 ; done" &
```

Comment:
so this is not a busy loop ?

Reply:
only if it were to crash constantly;  under normal operations, it starts and stays up until you kill it or the service has no endpoints (it seems)

Reply:
make sense to add a `sleep 1;` in case it exits immediately for some seconds?

Reply:
:checkered_flag: